web/yarn.lock

```diff
@@ -1655,6 +1655,9 @@
     "@webassemblyjs/wast-parser" "1.8.5"
     "@xtuc/long" "4.2.2"
 
+<<<<<<< HEAD
```

Comment:
What is going on in this file?

Reply:
Botched conflict resolving it would seem, I'll get it removed